images/gh-gl-sync/SpackCIBridge.py

```diff
@@ -15,6 +15,14 @@
 import urllib.parse
 import urllib.request
 
+if 'SENTRY_DSN' in os.environ:
+    import sentry_sdk
+
+    sentry_sdk.init(
+        dsn=os.environ['SENTRY_DSN'],
+        traces_sample_rate=0.01,
+    )
```

Comment:
```suggestion
import sentry_sdk

sentry_sdk.init(traces_sample_rate=0.1)
```

See https://docs.sentry.io/platforms/python/configuration/options/#dsn.

I bumped up the performance sampling rate too, I think 10% is reasonable for something that only runs ~12/hr.

Reply:
Is there any easy way to pass staging/prod as the environment?

Reply:
> Is there any easy way to pass staging/prod as the environment?

There might be, I have an idea I could try. but since we have the separate sentry instance for staging, why would we need to differentiate between them?

Reply:
🤦‍♂️ d'oh, nevermind.